api/src/test/java/com/code4ro/nextdoor/emergency/contact/service/EmergencyContactServiceImplTest.java

```diff
@@ -0,0 +1,96 @@
+package com.code4ro.nextdoor.emergency.contact.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import com.code4ro.nextdoor.emergency.contact.dto.EmergencyContactDto;
+import com.code4ro.nextdoor.emergency.contact.entity.EmergencyContact;
+import com.code4ro.nextdoor.emergency.contact.repository.EmergencyContactRepository;
+import com.code4ro.nextdoor.emergency.contact.service.impl.EmergencyContactServiceImpl;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
Poti anota atat clasa cat si metodele cu @DisplayName.
Si ar fi ok sa facem @DisplayName("Emergency contact service should) pe clasa si la fiecare metoda @DisplayName cu ce ar trebui sa se intample

Reply:
sure :D